### PR TITLE
[JENKINS-31098] Deletage remove a node back to Computer.doDoDelete()

### PR DESCRIPTION
**Pull request by pjanouse**
Please review. Thx Pavel

Note: Occurred exceptions during execution of DeleteNodeCommandTest aren't related to my impl. and are the same with the original.

http://issues.jenkins-ci.org/browse/JENKINS-31098

### CODE REVIEW

**Comment by olivergondza** [comment]
:+1:

**Comment by oleg-nenashev** [comment]
:+1:

**Comment by felfert** [comment]
:+1:
Tested also against 1.625.1: lgtm.

File: test/src/test/java/hudson/cli/DeleteNodeCommandTest.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
     @Before public void setUp() {
 
-        command = new CLICommandInvoker(j, new DeleteNodeCommand());
+        command = new CLICommandInvoker(j, "delete-node");
```
Comment (daniel-beck) [inline test/src/test/java/hudson/cli/DeleteNodeCommandTest.java]: What is this about?

Comment (olivergondza) [inline test/src/test/java/hudson/cli/DeleteNodeCommandTest.java]: It reflects my comment from the original PR. The effect is the same with the exception this resolved the command even when registered using `CLIMethod` (and therefore do not have a class). This is useful for refactoring `CLIMethod`s to `CLICommand`s as one can checkout implementation back and forth to run tests and ensure the refactoring have not changed the behaviour.

**Pull request merged**